src/secrets/components/SecretContextMenu.tsx

```diff
@@ -0,0 +1,69 @@
+// Libraries
+import React, {FC} from 'react'
+import CopyToClipboard from 'react-copy-to-clipboard'
+import {connect, ConnectedProps} from 'react-redux'
+
+// Components
+import {
+  Button,
+  ButtonShape,
+  ComponentColor,
+  ComponentSize,
+} from '@influxdata/clockface'
+
+// Actions
+import {notify as notifyAction} from 'src/shared/actions/notifications'
+
+// Notifications
+import {
+  copyToClipboardSuccess,
+  copyToClipboardFailed,
+} from 'src/shared/copy/notifications'
+
+// Types
+import {Secret} from 'src/types'
+import {Context} from 'src/clockface'
+
+interface OwnProps {
+  secret: Secret
+}
+
+type ReduxProps = ConnectedProps<typeof connector>
+type Props = OwnProps & ReduxProps
+
+const SecretContextMenu: FC<Props> = ({secret, notify}) => {
+  const handleCopyAttempt = (
+    copiedText: string,
+    isSuccessful: boolean
+  ): void => {
+    if (isSuccessful) {
+      notify(copyToClipboardSuccess(copiedText, 'Secret ID'))
+    } else {
+      notify(copyToClipboardFailed(copiedText, 'Secret ID'))
+    }
+  }
+
+  return (
+    <Context>
+      <CopyToClipboard text={secret.id} onCopy={handleCopyAttempt}>
```

Comment:
What's copied when we click the button?

Reply:
The value of `text` which is the key from the secret.

Reply:
Awww, yup didn’t see text being assigned the secret id. Cool cool cool